src/xhr.js

```diff
@@ -63,6 +69,57 @@ class Xhr {
 
 
 /**
+ * @param {?FetchInit=} opt_init
+ * @private
+ * @exportForTesting
+ */
+export function getMethod_(opt_init) {
+  let allowedMethods = ['GET', 'POST'];
+
+  console.log(JSON.stringify(opt_init));
+  if (opt_init === undefined || opt_init.method == null) {
```

Comment:
renamed this to `normalizeMethod_` and i only pass in the `method` w/o the whole init object